packages/composer/lib/root-endpoint/index.js

```diff
@@ -8,20 +8,67 @@ module.exports = async (app, opts) => {
   app.register(fastifyStatic, {
     root: join(__dirname, 'public'),
   })
-
+  app.register(require('@fastify/view'), {
+    engine: {
+      nunjucks: require('nunjucks')
+    },
+    root: join(__dirname, 'public')
+  })
   // root endpoint
   app.route({
     method: 'GET',
     path: '/',
     schema: { hide: true },
-    handler: (req, reply) => {
+    handler: async (req, reply) => {
       const uaString = req.headers['user-agent']
       if (uaString) {
         const parsed = userAgentParser(uaString)
         if (parsed.browser.name !== undefined) {
-          return reply.sendFile('./index.html')
+          const serviceTypes = {
+            proxy: {
+              title: 'Reverse Proxy',
+              icon: './images/reverse-proxy.svg',
+              services: []
+            },
+            openapi: {
+              title: 'OpenAPI',
+              icon: './images/openapi.svg',
+              services: []
+            },
+            graphql: {
+              title: 'GraphQL',
+              icon: './images/graphql.svg',
+              services: []
+            }
+          }
+          console.log(app.platformatic.config.composer.services, '@@@@@@@@@@@@@@@@@@@@')
+          app.platformatic.config.composer.services.forEach((s) => {
+            if (s.openapi) {
+              serviceTypes.openapi.services.push({
+                ...s,
+                externalLink: `/${s.id}`
```

Comment:
these should be relative links, otherwise it wouldn't work if there is a prefix added in front.